common/script/content.coffee

```diff
@@ -1919,7 +1919,7 @@ api.subscriptionBlocks =
   basic_12mo: months:12, price:48
 _.each api.subscriptionBlocks, (b,k)->b.key = k
 
-repeat = {m:true,t:true,w:true,th:true,f:true,s:true,su:true}
+# repeat = {m:true,t:true,w:true,th:true,f:true,s:true,su:true}
```

Comment:
Not exactly. It was used only by the Dailies lines a bit below that (default Dailies for new accounts) and those lines were commented out in recent weeks, so I've commented out the `repeat` for consistency.